templates/admin/sources/person/change_form.html

```diff
@@ -0,0 +1,50 @@
+{% extends 'admin/change_form.html' %}
+{% load static %}
+
+{% block content_title %}
+  {% if request.GET.edit %}
+    <h1>Edit {{ opts.verbose_name }}</h1>
+  {% else %}
+    <h1>View {{ opts.verbose_name }}</h1>
+  {% endif %}
+{% endblock %}
+
+{% block object-tools-items %}
+
+  <style>
+    .object-tools a:link.change-link, .object-tools a:visited.change-link {
+      background: #417690;
+    }
+
+    .object-tools .change-link img {
+      margin-right: 3px;
+      padding-bottom: 2px;
+    }
+  </style>
```

Comment:
Sorry, I should have been more specific before. We can put these in a separate file and then load them. Something like the following in `change_form.html`:

`{% block extrastyle %} <link rel="stylesheet" type="text/css" href="{% static 'css/admin/css' %}"> {% endblock %}`

Also, I'm just noticing we have two `change_form.html` files, so we should see about combining those.

Reply:
Hey sorry style tag isn't being applied anymore, think I didn't refresh correctly when I deleted the css file.

Reply:
disregard...for comment below

Reply:
Will update css.

The second change form is specific to the Person modal. We seem to follow this format on divesite. Do you only want one change_form for the whole site?

Reply:
@eric38383 Ah, gotcha. Sorry, I should have looked at things more closely. We should keep both!

Reply:
Will update css.

The second change form is specific to the Person modal. We seem to follow this format on divesite. Do you only want one change_form for the whole site?

Reply:
@greglinch Sorry keep putting comments in the wrong places ...anyways. The block extrastyle tag seems to overwrite some css so i used media class in the admin class for the css file